RestaurantsNearMe/Data/CloudKitServiceError.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CloudKitServiceError.swift
+//  RestaurantsNearMe
+//
+//  Created by Colin Evans on 2023-11-27.
+//
+
+import Foundation
+
+enum CloudKitServiceError: Error {
+  case containerNotAvailable
+  case fetchFailed
+  case incorrectDataFormat
+}
+
+extension CloudKitServiceError {
+  func toErrorString() -> String {
```

Comment:
Any reason not to use `CustomStringConvertible` or `CustomDebugStringConvertible` here instead?